pkg/api/v1/types.go

```diff
@@ -284,11 +284,31 @@ type PersistentVolumeSpec struct {
 	// ClaimRef is expected to be non-nil when bound.
 	// claim.VolumeName is the authoritative bind between PV and PVC.
 	ClaimRef *ObjectReference `json:"claimRef,omitempty" description:"when bound, a reference to the bound claim"`
+	// PersistentVolumeReclaimPolicy represents what happens to a persistent volume when released from its claim.
+	PersistentVolumeReclaimPolicy PersistentVolumeReclaimPolicy `json:"persistentVolumeReclaimPolicy,omitempty" description:"persistentVolumeReclaimPolicy is what happens to a volume when released from its claim; Default is Retain."`
```

Comment:
The descriptions don't generally name the fields.

`what happens to a volume when released from its claim; default is Retain`